doc/releases/nautilus.rst

```diff
@@ -1,3 +1,153 @@
+v14.2.12 Nautilus
+=================
+
+This is the 12th backport release in the Nautilus series. This release
+brings a number of bugfixes across all major components of Ceph. We recommend
+that all Nautilus users upgrade to this release.
+
+Notable Changes
+---------------
+
+* The ``ceph df`` command now lists the number of pgs in each pool.
+
+* Monitors now have a config option ``mon_osd_warn_num_repaired``, 10 by default.
+  If any OSD has repaired more than this many I/O errors in stored data a
+  ``OSD_TOO_MANY_REPAIRS`` health warning is generated.  In order to allow
+  clearing of the warning, a new command ``ceph tell osd.# clear_shards_repaired [count]``
+  has been added.  By default it will set the repair count to 0.  If you wanted
+  to be warned again if additional repairs are performed you can provide a value
+  to the command and specify the value of ``mon_osd_warn_num_repaired``.
+  This command will be replaced in future releases by the health mute/unmute feature.
+
+* It is now possible to specify the initial monitor to contact for Ceph tools
+  and daemons using the ``mon_host_override`` config option or
+  ``--mon-host-override <ip>`` command-line switch. This generally should only
+  be used for debugging and only affects initial communication with Ceph's
+  monitor cluster.
+
```

Comment:
Can we please add a note similar this https://github.com/ceph/ceph/pull/37109/files#diff-52fc1d32fab56fed8f65b5e2db4f9ea3e3e7240f4b7797ef2b1a0b6dcb5c3a1bR25-R27 and reference https://tracker.ceph.com/issues/47296 and https://github.com/ceph/ceph/pull/36982 instead

Reply:
Thanks, @neha-ojha - I have made the change you requested.

Reply:
Thanks @smithfarm!